.gitignore

```diff
@@ -26,3 +26,4 @@ bin
 charts/*.tgz
 kubesphere.yaml
 */**/*.xml
+testbin/
```

Comment:
Please add an empty line at the end of file.

Reply:
Okay.